web/docs/migrate-from-0-11-to-0-12.md

```diff
@@ -302,16 +302,6 @@ directory `foo`, you should:
 That's it! You now have a properly structured Wasp 0.12.0 project in the `foo` directory.
 Your app probably doesn't quite work yet due to the breaking changes in Auth, which we will migrate next.
 
-:::tip
```

Comment:
I removed this tip because we talk about this in Next Steps anyway, and we don't really want them to start changing the structure yet, because they need to yet migrate the Auth and will just make a situation more complex if they do this now, so better not to mention it, both to avoid additional cognitive load and to avoid them possibly doing it now.